packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -706,6 +706,89 @@ skipInNode(describe)('plugin-meetings', () => {
           throw e;
         }));
 
+      it('alice resolution is 1080p, check what bob receives', (done) => {
+        testUtils.delayedPromise(alice.meeting.setLocalVideoQuality('1080p'))
+          .then(() => {
+            testUtils.waitUntil(2000);
+            const remoteSettings = bob.meeting.mediaProperties.remoteVideoTrack.getSettings();
+            const localSettings = alice.meeting.mediaProperties.videoTrack.getSettings();
+
+            assert.equal(localSettings.height, '1080');
+            assert.equal(remoteSettings.height, '1080');
+            done();
+          }).catch((error) => {
```

Comment:
why are we catching the errors? this will cause the test to always pass even if something (like the asserts) throws, won't it?

Reply:
Removed.